forms/offer/products/OfferTextileProduct.tsx

```diff
@@ -69,10 +69,13 @@ export const OfferTextileProduct: FC<IProps> = ({
   })
 
   const onFormSubmit = (values: DonateItemRequestWithoutName) => {
-    const donateItemRequest: DonateItemRequestWithoutName = { ...values }
+    const donateItemRequest: DonateItemRequest = {
+      ...values,
+      name: t('signup.products.textile'),
```

Comment:
this is going to be submitted to backend, which is not desired

Reply:
please check #304. I rebased that PR so it's mergeable.